rllib/core/learner/tf/tf_learner.py

```diff
@@ -119,6 +115,43 @@ def postprocess_gradients(
             )
         return gradients_dict
 
+    @override(Learner)
+    def load_state(
+        self,
+        dir: Union[str, pathlib.Path],
+    ) -> None:
+        # this is a potentially very expensive operation since we are creating an
+        # MARL Module at build time then destroying it and creating a new one
+        # from checkpoint, but it is necessary because of complications with
```

Comment:
I think you should elaborate on what complication you are talking about here.

Reply:
done